packages/cli/src/errors/feature-not-licensed.error.ts

```diff
@@ -3,6 +3,8 @@ import { ApplicationError } from 'n8n-workflow';
 import type { LICENSE_FEATURES } from '@/constants';
 
 export class FeatureNotLicensedError extends ApplicationError {
+	stack = ''; // not an error to debug
```

Comment:
Could we instead set a `level` to `warning` here, and then skip logging the stack traces for any warning?

Reply:
Is `FeatureNotLicensedError` warning level? If you enable multi-main without a license, we throw this and prevent the instance from starting up. It'd be odd if the logs for a blocked startup show a warning instead of an error?

Reply:
do we want this error to go to Sentry? 
I get that most multi-main users possibly won't have multi-main setup, but some might. If we keep this as an error, the crash-looping instances will spam Sentry.

Reply:
Good point, done. Oddly this error is used in the `isLicensed` middleware so I'd expect this to show up in Sentry, but it's not there at all.